config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/SsgActiveConnectorEntityBuilder.java

```diff
@@ -0,0 +1,147 @@
+package com.ca.apim.gateway.cagatewayconfig.bundle.builder;
+
+import com.ca.apim.gateway.cagatewayconfig.beans.*;
+import com.ca.apim.gateway.cagatewayconfig.util.IdGenerator;
+import com.ca.apim.gateway.cagatewayconfig.util.entity.EntityTypes;
+import org.jetbrains.annotations.NotNull;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import javax.inject.Inject;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.stream.Stream;
+
+import static com.ca.apim.gateway.cagatewayconfig.environment.EnvironmentBundleUtils.getDeploymentBundle;
+import static com.ca.apim.gateway.cagatewayconfig.util.entity.EntityTypes.LISTEN_PORT_TYPE;
+import static com.ca.apim.gateway.cagatewayconfig.util.entity.EntityTypes.SSG_ACTIVE_CONNECTOR;
+import static com.ca.apim.gateway.cagatewayconfig.util.gateway.BuilderUtils.buildAndAppendPropertiesElement;
+import static com.ca.apim.gateway.cagatewayconfig.util.gateway.BundleElementNames.*;
+import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.createElementWithAttribute;
+import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.createElementWithTextContent;
+import static java.lang.Boolean.TRUE;
+import static java.util.stream.Collectors.toList;
+
+public class SsgActiveConnectorEntityBuilder implements EntityBuilder {
+    private static final Integer ORDER = 1201;
```

Comment:
We are following multiple of 100 for ORDER. This could be 1600 or 1700 I suppose.

Reply:
OK